core/devmode/src/test/java/io/quarkus/dev/CompilerFlagsTest.java

```diff
@@ -0,0 +1,86 @@
+package io.quarkus.dev;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Can you please replace with the regular imports? We try to not use start imports

Reply:
Done.